src/v0/destinations/braze/identityResolutionUtils.test.ts

```
@@ -0,0 +1,1322 @@
+describe('identityResolutionUtils', () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+    
+    // Default mock implementations
+    mockedGetEndpointFromConfig.mockReturnValue('https://rest.iad-03.braze.com');
+    mockedGetIdentifyEndpoint.mockReturnValue('https://rest.iad-03.braze.com/users/identify');
+    mockedGetDynamicErrorType.mockReturnValue('retryable');
+    mockedIsHttpStatusSuccess.mockImplementation((status) => status >= 200 && status < 300);
+    
+    // Mock IDENTIFY_BRAZE_MAX_REQ_COUNT
+    (IDENTIFY_BRAZE_MAX_REQ_COUNT as any) = 50;
```

Biome (1.9.4)

[error] The imported variable IDENTIFY_BRAZE_MAX_REQ_COUNT is read-only

The variable is imported here

Use a local variable instead of reassigning an import.

(lint/suspicious/noImportAssign)